ts/Core/Legend/Legend.ts

```diff
@@ -482,16 +488,22 @@ class Legend {
             Series|Point
         )
     ): void {
-        const checkbox = item.checkbox;
+        const checkbox = item.checkbox,
+            legendData = item.legendData || {};
 
         // destroy SVG elements
-        ['legendItem', 'legendLine', 'legendSymbol', 'legendGroup'].forEach(
+        ['legendLine'].forEach(
             function (key: string): void {
                 if ((item as any)[key]) {
                     (item as any)[key] = (item as any)[key].destroy();
                 }
             }
         );
```

Comment:
Is this used? I thought `item.legendLine` as moved to `legendData.line`? And if so, we should add `line` to the array at line 502.